docs-gen/content/rule_set/data_entry/_index.md

```diff
@@ -13,9 +13,9 @@ a publisher (or producer) that continuously updates the signal value while an
 attribute has a set value, defined in the specification, that never changes.
 As summary, besides [```branch```](/rule_set/branches) type can be:
 
-* **```attribute```**, which describes static read-only value.
-* **```sensor```**, which describes non-static read-only signal.
-* **```actuator```**, which describes a non-static read-write signal.
+* **```attribute```**: attributes are not expected to change once they're set (e.g. vehicle identification number)
```

Comment:
Good summary I think.
In a complementary "system design" description we might clarify that this is either done by being set by a certain system, or by "software-download" of a configuration file, etc.  But what you write here is a short and valid description I think.